ci-operator/config/medik8s/fence-agents-remediation/medik8s-fence-agents-remediation-main__4.12.yaml

```diff
@@ -28,25 +34,45 @@ resources:
       cpu: 500m
       memory: 1000Mi
 tests:
+- as: test
+  commands: make test
+  container:
+    from: src
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
 - as: openshift-e2e
   skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
   steps:
     cluster_profile: aws
+    dependencies:
+      OO_BUNDLE: my-bundle
```

Comment:
In other operators (e.g. NHC) the dependency is declared this way

```
dependencies:
- env: OO_BUNDLE
  name: my-bundle
```

are you sure this is correct as well?

Reply:
Yes, I got this recommendation after communicating in [`forum-ocp-testplatform`](https://redhat-internal.slack.com/archives/CBN38N3MW/p1696253793949649).

Reply:
It doesn't add the label from here :(